packages/sit-onyx/src/components/OnyxRadioButton/OnyxRadioButton.stories.ts

```diff
@@ -15,6 +15,7 @@ const meta: Meta<typeof OnyxRadioButton> = {
       isLoading: { type: "boolean" },
       isDisabled: { type: "boolean" },
       selected: { type: "boolean" },
+      errorMessage: { type: "string" },
```

Comment:
The Storybook props table is missing all descriptions and prop types because an union type is used. This is a known Storybook limitation currently. I'd prefer to remove the union type for now

![image](https://github.com/SchwarzIT/onyx/assets/67898185/6972a236-38b0-4818-b60b-60023f35f906)

Reply:
That sucks :( Do you have an issue link? Any idea when this will be fixed?
Because if this will be a long running issue, I would prefer having great typing over duplicated docs.

Reply:
I added a follow-up ticket: https://github.com/orgs/SchwarzIT/projects/5/views/7?pane=issue&itemId=53102909